test/perf/perf_simple_query.cc

```diff
@@ -105,12 +107,17 @@ std::ostream& operator<<(std::ostream& os, const test_config& cfg) {
 
 static void create_partitions(cql_test_env& env, test_config& cfg) {
     std::cout << "Creating " << cfg.partitions << " partitions..." << std::endl;
+    unsigned next_flush = (cfg.memtable_partitions > 0 ? cfg.memtable_partitions : cfg.partitions);
     for (unsigned sequence = 0; sequence < cfg.partitions; ++sequence) {
         if (cfg.counters) {
             execute_counter_update_for_key(env, make_key(sequence));
         } else {
             execute_update_for_key(env, make_key(sequence));
         }
+        if (sequence + 1 >= next_flush) {
```

Comment:
Do we need that also after exiting the loop in case cfg.partitions is not divisble by cfg.memtable_partitions?
If so, this can be done in the beginning of the loop `if (sequence >= next_flush)`

Reply:
Otherwise, LGTM